securedrop/management/run.py

```diff
@@ -151,12 +154,15 @@ def run():
 
     """
     procs = [
-        DevServerProcess('Source Interface',
-                         ['python', 'source.py'],
-                         'blue'),
-        DevServerProcess('Document Interface',
-                         ['python', 'journalist.py'],
-                         'cyan'),
+        lambda: DevServerProcess('Source Interface',
+                                 ['python', 'source.py'],
+                                 'blue'),
+        lambda: DevServerProcess('Document Interface',
+                                 ['python', 'journalist.py'],
+                                 'cyan'),
+        lambda: DevServerProcess('SASS Compiler',
+                                 ['sass', '--watch', 'sass:static/css'],
+                                 'magenta'),
```

Comment:
@heartsucker Thanks for the explanation! I'll take a look at this myself - I think there might be a way to more gracefully handle that case within DevServerProcess.